src/main/java/com/byclosure/jenkins/plugins/gcloud/GCloudBuildWrapper.java

```diff
@@ -51,11 +102,25 @@ public boolean tearDown(AbstractBuild build, final BuildListener listener) throw
 				serviceAccount.cleanUp();
 				return true;
 			}
-
 		};
 	}
 
-	public String getCredentialsId() {
+    public GCloudInstallation getSDK() {
+        GCloudInstallation[] installations = GCloudInstallation.getInstallations();
+        if (installations.length == 1) return installations[0];
```

Comment:
There's a risk of returning a wrong installation (e.g. if a user use different versions). I would vote against the implicit fallback